samples/trafficmgmt-namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: install.istio.io/v1alpha2
```

Comment:
can we use traffiicmanagement instead of trafficmgmt

Reply:
Done.